fic/resource_eri_port_to_azure_private_connection_v1.go

```diff
@@ -0,0 +1,303 @@
+package fic
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	gofic "github.com/nttcom/go-fic"
+	connections "github.com/nttcom/go-fic/fic/eri/v1/port_to_azure_private_connections"
+)
+
+func resourceEriPortToAzurePrivateConnectionV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEriPortToAzurePrivateConnectionV1Create,
+		Read:   resourceEriPortToAzurePrivateConnectionV1Read,
+		Delete: resourceEriPortToAzurePrivateConnectionV1Delete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"source_primary_port_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"source_primary_vlan": &schema.Schema{
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"source_secondary_port_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"source_secondary_vlan": &schema.Schema{
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"source_asn": &schema.Schema{
```

Comment:
Could it be that API specification of FIC is wrong?
In API specification, ASN is defined as number format.
But this parameter is defines as string both in FIC provider and go-fic.

Reply:
To accordance with the API document:

https://fic.ntt.com/documents/api-references/connection-azure/rsts/Port%20to%20Azure(MS).html

, actual type of the parameter is string. The test code is successfully done with it in fact.
Therefore, I think this is an document bug:|